lib/ProductOpener/Display.pm

```diff
@@ -6923,8 +6923,30 @@ sub display_field($$) {
 			$value =~ s/\n/<br>/g;
 		}
 	}
+	
+	my $to_do_status = '';
+	my $done_status = '';
+
+	if ($field eq 'states'){
+		my @to_do_status = '';
```

Comment:
That's why your array has an extra element, this assigns ''  (an empty string) to the first element of the array.

To create an empty array:

my @to_do_status = ();